lib/common/src/DevMode.php

```diff
@@ -51,6 +51,15 @@ public static function hasExemptionForNode(DOMNode $node)
             return false;
         }
 
+        /**
+         * @var Document $document
+         */
```

Comment:
I don't think this is required because it will only be a `Document` per below:

```suggestion
```

Reply:
This is just to make `phpstan` happy. It incorrectly detects `$document` here as a `DOMDocument`,which would not have the `hasInitialAmpDevMode` method.

Reply:
But it shouldn't be needed now that you did 6314af6.

Reply:
I found what was missing. Fixed in 472c6fc1c.